frontend/www/js/omegaup/components/identity/Editv2.vue

```diff
@@ -103,69 +103,69 @@ import Autocomplete from '../Autocomplete.vue';
   },
 })
 export default class IdentityEdit extends Vue {
-  @Prop() identity!: omegaup.Identity;
+  @Prop({ default: null }) identity!: omegaup.Identity;
   @Prop() countries!: iso3166.Country[];
 
   T = T;
   typeahead = typeahead;
-  username = this.identity.username;
-  name = this.identity.name;
-  gender = this.identity.gender;
-  school = this.identity.school;
-  schoolId = this.identity.school_id;
-  selectedCountry = this.identity.country_id ?? 'MX';
-  selectedState = this.identity.state_id;
+  selectedIdentity = Object.assign(
+    <omegaup.Identity>{
+      username: '',
+      classname: '',
+      name: '',
+      gender: '',
+      school: '',
+      school_id: 0,
+      country_id: '',
+      state_id: '',
+    },
+    this.identity,
+  );
 
   @Watch('identity')
   onIdentityChanged(newIdentity: omegaup.Identity) {
-    this.username = newIdentity.username;
-    this.name = newIdentity.name;
-    this.gender = newIdentity.gender;
-    this.school = newIdentity.school;
-    this.schoolId = newIdentity.school_id;
-    this.selectedCountry = newIdentity.country_id ?? 'MX';
-    this.selectedState = newIdentity.state_id;
+    Object.assign(this.identity, newIdentity);
   }
 
   @Watch('selectedCountry')
   onPropertyChanged(newContry: string) {
     if (this.identity.country_id == newContry) {
-      this.selectedState = this.identity.state_id;
+      this.selectedIdentity.state_id = this.identity.state_id;
     } else {
-      this.selectedState = Object.keys(this.countryStates)[0].split('-')[1];
+      this.selectedIdentity.state_id = Object.keys(this.countryStates)[0].split(
+        '-',
+      )[1];
     }
   }
 
   get groupName(): string {
-    if (typeof this.identity === 'undefined') {
-      return '';
-    }
-    return `${this.identity.username.split(':')[0]}`;
+    return `${this.selectedIdentity.username.split(':')[0]}`;
```

Comment:
```suggestion
    return this.selectedIdentity.username.split(':')[0];
```

esto ya es una cadena, así que no necesita la cadenificación.

Reply:
Listo